frontend/components/queries/QueryPageWrapper/helpers.js

```diff
@@ -14,6 +14,12 @@ export const fetchQuery = (dispatch, queryID) => {
       errorMessage = "The query you requested does not exist in Fleet.";
     }
 
+    // LEGACY CODE - I had to do this :-( MP 1/25/22
```

Comment:
Consider adding a quick explanation of why you had to do this like you did in the PR comments. Otherwise it will be a mystery in the future :)